packages/rpc-api/src/getInflationReward.ts

```diff
@@ -23,23 +26,25 @@ type GetInflationRewardApiConfig = Readonly<{
 }>;
 
 type InflationReward = Readonly<{
-    // Reward amount in lamports
+    /** Reward amount in {@link Lamports} */
     amount: Lamports;
-    // Vote account commission when the reward was credited
+    /** Vote account commission when the reward was credited */
     commission: number;
-    // The slot in which the rewards are effective
+    /** The slot in which the rewards are delivered */
```

Comment:
I _think_ it's more to the point to use the word ‘delivered’ here, but someone check me for accuracy.

Reply:
Agreed, makes more sense to me!